docs/content/docs/connectors/datastream/formats/csv.md

```diff
@@ -113,6 +115,22 @@ CsvReaderFormat<ComplexPojo> csvFormat =
                         .build(),
                 TypeInformation.of(ComplexPojo.class));
 ```
+
+For PyFlink users, a csv schema can be defined by manually adding columns, and the output type of the csv source will be a Row with each column mapped to a field.
+```python
+schema = CsvSchema.builder() \
+    .add_number_column('id', number_type=DataTypes.BIGINT()) \
```

Comment:
Why not use `TypeInformation`, but `DataType`？

Reply:
Because I'm basically reusing the code implemented for Table API, where interfaces are designed for DataType.